### ISSUE TITLE
Package local formula

### ISSUE THREAD

**Issue by W4RH4WK** [timsutton/brew-pkg#7]
I have installed encfs using a local formula with the following command:

    $ brew install contrib/encfs.rb

but I am not able to package this using brew-pkg:

    $ brew pkg --with-deps contrib/encfs.rb
    ==> Creating package staging root using Homebrew prefix /usr/local
    Error: No available formula with the name "encfs"

**Comment by timsutton** [comment]
Have you tried just `brew pkg --with-deps encfs`? If I install encfs from the existing brew repo this command works. I should think it would work the same way even if it was installed from a local formula.

**Comment by W4RH4WK** [comment]
Yes, I have tried this but it didn't work either. I get the same error in this case. Seems like brew searches for a formula in its repository / taps (?).

    $ brew pkg --with-deps encfs
    Error: No available formula with the name "encfs"

**Comment by timsutton** [comment]
I can see the use case for this, so if you feel like digging into what changes would be needed for local formulae to also work, that would be great!

**Comment by W4RH4WK** [comment]
I'll let you know if I come up with something, but no promises.
